advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -143,9 +143,7 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
         "Bower" -> null
         "Composer" -> Ecosystem.PACKAGIST
         "Crate" -> Ecosystem.CRATES_IO
-        // TODO: The entries for Go should be merged, see https://github.com/oss-review-toolkit/ort/issues/5533.
-        "GoDep" -> Ecosystem.GO
-        "GoMod" -> Ecosystem.GO
+        "Go" -> Ecosystem.GO
```

Comment:
I am not sure whether this is used in the context of parsing of old reports, but just to mention it: you could also keep the "old" mapping for more backwards compatibility